frontend/src/Components/CreateCommentComponent.tsx

```diff
@@ -65,17 +67,24 @@ export const CreateCommentComponentRestricted = observer((props: CreateCommentPr
     }}/>;
 });
 
+const Item = (item: { entity: string }) => {
+    return <div>{`${item.entity}`}</div>;
+};
+let answer: HTMLTextAreaElement | undefined = undefined;
+
 export default function CreateCommentComponent(props: CreateCommentProps) {
-    const answerRef = useRef<HTMLTextAreaElement>(null);
+    const answerRef = useRef<HTMLTextAreaElement>();
+    if (!answer) {
```

Comment:
Dead code!

Also, please use `answerRef` instead of `answer`. React components should be self-contained.